main/src/library/Console.flix

```diff
@@ -15,42 +15,70 @@
  */
 
 namespace Console {
+    use Option.flatMap;
+    use Object.isNull;
 
     ///
-    /// Alias for `Console.StdIn.readLine`.
+    /// Wrapper for `java.lang.System.Console().readLine()`.
+    /// Returns `None` if a there is no line to read or no console is available.
     ///
-    pub def readLine(): Option[String] & Impure = Console/StdIn.readLine(())
+    pub def readLine(): Option[String] & Impure =
+        import java.io.Console.readLine();
+        let* c = getConsole();
+        let line = readLine(c);
+        if (isNull(line)) None else Some(line)
 
     ///
-    /// Alias for `Console.StdOut.print`.
+    /// Returns true if a console is available.
     ///
-    pub def print(s: String): Unit & Impure = Console/StdOut.print(s)
+    pub def hasConsole(): Bool & Impure =
+        not Option.isEmpty(getConsole())
 
     ///
-    /// Alias for `Console.StdOut.printLine`.
+    /// Wrapper for `java.lang.System.console().writer().print(s)`.
+    /// Returns `Err()` if no console is available.
     ///
-    pub def printLine(s: String): Unit & Impure = Console/StdOut.printLine(s)
+    pub def print(s: String): Unit & Impure =
+        import java.io.Console.writer();
+        import java.io.PrintWriter.print(String);
+        match getConsole() {
+            case None => ()
+            case Some(c) =>
+                print(writer(c), s)
+        }
 
     ///
-    /// Operations on the Standard Input Stream.
+    /// Wrapper for `java.lang.System.console().writer().println(s)`.
+    /// Returns `Err()` if no console is available.
     ///
-    namespace StdIn {
```

Comment:
Not sure why this namespace was lost.

I think we should preserve the same structure with `StdIn/StdOut` etc. If you want to change it, we can do it separately.

Reply:
I removed it since console is now specifically not stdIn